rules-java-ee/tests/src/test/resources/discover-spring-remote-services/RestController.java

```diff
@@ -0,0 +1,72 @@
+package com.redhat.rest;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.hateoas.Resource;
+import org.springframework.hateoas.Resources;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class EmployeeController {
```

Comment:
Shouldn't the file name of this class be EmployeeController? I believe test resource classes don't need to compile so it probably doesn't matter.

Reply:
+1